setup.cfg

```
@@ -32,6 +32,7 @@ filterwarnings =
     ignore:The distutils(.+) is deprecated:DeprecationWarning
     # pkg_resources
     ignore::DeprecationWarning:pkg_resources
+    ignore:pkg_resources is deprecated as an API:DeprecationWarning:__main__
```

https://setuptools.pypa.io/en/latest/pkg_resources.html
```suggestion
    ignore:pkg_resources is deprecated
```